VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/linked-participant-panel-model.ts

```diff
@@ -1,11 +1,16 @@
 import { ParticipantStatus, Role } from 'src/app/services/clients/api-client';
 import { HearingRole } from './hearing-role-model';
+import { IndividualPanelModel } from './individual-panel-model';
 import { PanelModel } from './panel-model-base';
 
 export class LinkedParticipantPanelModel extends PanelModel {
-    public participants: PanelModel[] = [];
+    public participants: IndividualPanelModel[] = [];
```

Comment:
This can be a participant or endpoint. Changed to this to prevent any chance of  a LinkedParticipantPanelModel having a LinkedParticipantPanelModel which could cause issues.

Reply:
Just to expand on this slightly as I was asked. I didn't want there to be any chance of 2 LinkedPanelModels to reference each other as that would cause a circular reference. I don't think that's likely to happen but I think it's better to be as specific as possible to prevent confusion. I initially thought this could be ParticipantPanelModel[] but endpoints can have an advocate as a linked participant so I needed to include them.

I'm open to name suggestions as I realise individual has other meanings elsewhere.